src/qibosoq/components.py

```diff
@@ -47,8 +47,8 @@ class Pulse:
     """Amplitude factor, multiplied by maximum gain of the DAC."""
     relative_phase: int
     """Relative phase (degrees)."""
-    start: float = field(compare=False)
-    """Start time (us)."""
+    start_delay: float = field(compare=False)
```

Comment:
I don't plan it, but it's actually a possibility...
Also, for Qick the delay is "by definition" the one between one pulse end and another pulse start so I wanted to emphasize it...
I didn't want to directly implement a end_delay because it kinda removes the possibility to have pulses "overlapping" on different channels